### DISCUSSION TITLE
【任务】2023年1-2月社区激励任务｜50,000 $PEOPLE

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#399] [任务 | Bounty]
WTF Academy 社区激励任务，包括但不限于审核，改善WTF极简教程，也包括日常的PR/issue。这部分奖励优先分配给月度社区激励任务，剩余分给当月贡献者。

## 任务 1：翻译 Ethers.js v6文档（beta）

Ethers.js，使用最多的以太坊js库，将在2月发布v6版本，一群热心的志愿者决定将它翻译成中文。目前v6文档还没有完全写完，所以咱们只翻译比较完整的章节。内容不多，因为最长的章节（Application Programming Interface）还没出。

**要求**:
- 英文熟练，看过并看懂ethers v5版本的英文文档: https://docs.ethers.org/v5/api/
- 熟悉 ethers.js，阅读过WTF-Ethers教程
- 使用过chatgpt: 为了提高效率，先用chatgpt机翻，然后人工校对。
- 熟悉markdown语法

**赏金**：5,000 $PEOPLE 代币，根据工作量分配。

认领情况：

1. Doc首页
@JustinAsdz 0x3Da4da2De7a937aEbBe26166B4f06EDE7Bc77BE5 1000

2. Getting Started：相对内容多一些，2个人协作翻译。
@beihaili beihai1337.eth 1000
@pseudoyu 0xb461c1521eE9D96a9C950337f0851b79bD66CAe1 1000
3. Migrating from v5： 内容不多，但是很重要 
0xAA认领 wtfacademy.eth 1000

4. Contributing and Hacking
@seasidejuvenile666  0xC7AF21c54344ecD042c40286b05bd64BD8612dEE 1000

资源
- v6文档: https://docs.ethers.org/v6-beta/
- 中文翻译repo: https://github.com/WTFAcademy/WTF-ethers-v6-doc-cn

## 任务 2：ETH Beijing官网

WTF Academy将和PKU Blockchain一起主办ETH Beijing黑客松，帮助中文Builder建立第一个项目。我们现在需要一个简单的官网用于宣传。

### 要求
- 网页参考 ETH Tokyo 官网： https://ethglobal.com/events/tokyo
- 设计（一名）: 有网站设计经验，熟练运用figma
- 前端:（一名） 能快速完成静态网页的建立。

### 赏金

10000 $PEOPLE 代币，根据工作量分配。

### 认领情况

1. 设计： @tankxu
2. 前端: @cheng521521 
### 资源
- 宣传: https://twitter.com/0xAA_Science/status/1627848229986639873?s=20
- 赛道: https://twitter.com/0xAA_Science/status/1615709271261577216?s=20

## 任务 3：ETH Beijing log设计

让设计师们帮 ETH Beijing 黑客松设计 logo！

### 要求
发挥想象力，融合 eth 和 beijing 的元素和风格。
截止时间: 2.26
参与方法: 将作品回复到这条推特下: https://twitter.com/0xAA_Science/status/1628817818941812737?t=QPL_Byb2Jc2_Ybar8Y3v_w&s=19
### 赏金
5000 $PEOPLE，平均分配给前被选中的2个作品。

### 素材

可以参考素材发挥，也可以不参考

![20230221_094748.jpg](https://user-images.githubusercontent.com/14728591/220990579-8f045165-9a8b-476e-8796-cdec6f57caa3.jpg)

奖励已发放:
只有任务1发的People（映射到BSC），其余任务发的u（之前已经发了）。
txHash: https://bscscan.com/tx/0x4c58da06f7f80d1d717a3a7467b495aed1de495a7c0bdc6a2599d3f187624607

**Comment by seasidejuvenile666** [comment]
认领4.Contributing and Hacking

**Reply by AmazingAng** [reply]
好的 新年快乐

**Reply by seasidejuvenile666** [reply]
> 好的 新年快乐

新年快乐♥

**Comment by beihaili** [comment]
认领2.Getting Started的一部分

**Reply by beihaili** [reply]
P.S.我认领前一半吧@pseudoyu

**Comment by pseudoyu** [comment]
认领2.Getting Started的一部分

**Comment by JustinAsdz** [comment]
1 首页

**Comment by cheng521521** [comment]
接前端开发

**Reply by AmazingAng** [reply]
好的

**Comment by ychow** [comment]
我也想认领下前端开发的。

**Reply by AmazingAng** [reply]
可以来玩 不一定有任务 你微信多少？

**Comment by quinn-gaoo** [comment]
前端开发部分还需要人不

**Reply by AmazingAng** [reply]
可以来玩 不一定有任务 你微信多少？

**Reply by quinn-gaoo** [reply]
好的， Quinn-Gao

**Comment by chiguayeshao** [comment]
前端 认领 ETH Beijing官网

**Reply by AmazingAng** [reply]
可以来玩 不一定有任务 你微信多少？

**Reply by chiguayeshao** [reply]
18061982295

**Comment by Silence-dream** [comment]
前端开发部分还需要人不,我也想参与一些

**Reply by Silence-dream** [reply]
Wechat LoongSilence

**Reply by AmazingAng** [reply]
> Wechat LoongSilence

新的前端任务 https://github.com/AmazingAng/WTF-Solidity/discussions/435

**Comment by mssp299** [comment]
翻译这种活可以找我啊，我翻译出版过好几本IT图书了

**Reply by AmazingAng** [reply]
新的翻译任务 https://github.com/AmazingAng/WTF-Solidity/discussions/435

**Comment by jessecaii** [comment]
翻译or前端认领

**Reply by AmazingAng** [reply]
新的翻译和前端任务 https://github.com/AmazingAng/WTF-Solidity/discussions/435
